internal/network/service.go

```diff
@@ -60,6 +60,8 @@ func ReconcileInputService(er record.EventRecorder, k8sClient client.Client, nam
 		constants.AnnotationServingCertSecretName: certSecretName,
 	}
 
+	desired.Labels["component"] = "http-input-service"
```

Comment:
It would be nice to make "component" and "http-input-service" constants and use them here and in labels.Parse().

Reply:
Done